### ISSUE TITLE
ImportError: No module named 'backports' with Python 3.4

### ISSUE THREAD

**Issue by zanpeeters** [ipython/ipython#9416]
Just upgraded IPython to 4.2.0 on Python 3.4 and starting from the terminal fails:
```python
> ipython
Traceback (most recent call last):
  File "/opt/local/bin/ipython", line 4, in <module>
    from IPython import start_ipython
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/IPython/__init__.py", line 48, in <module>
    from .core.application import Application
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/IPython/core/application.py", line 24, in <module>
    from IPython.core import release, crashhandler
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/IPython/core/crashhandler.py", line 28, in <module>
    from IPython.core import ultratb
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/IPython/core/ultratb.py", line 121, in <module>
    from IPython.utils.terminal import get_terminal_size
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/IPython/utils/terminal.py", line 27, in <module>
    import backports.shutil_get_terminal_size
ImportError: No module named 'backports'
```

IPython/utils/terminal.py tries to `import backports.shutil_get_terminal_size` which is missing. I am using Python 3.4 and backports is not needed, shutil can be used directly.

If I replace "backports.shutil_get_terminal_size" with "shutil" on lines 9 and 125 in terminal.py, everything works fine for me. This should be wrapped in a "if PY2" or added to py3compat.py.

Other relevant information:

OS: Mac OS X 10.6.8
Python version: 3.4.4
IPython version: 4.2.0
Installer: MacPorts

I install IPython through MacPorts (`port install py34-ipython`). In the Portfile, backports-shutil_get_terminal_size is added as a dependency, but only for Python 2.7, which is appropriate.

**Comment by minrk** [comment]
This is a macports bug: it has failed to express a dependency on `backports.shutil_get_terminal_size`, which is a dependency of IPython 4.2.

**Comment by minrk** [comment]
#9417 updates the backport dependency to only apply on Python 2, and use shutil directly on Python 3. But the ports package for 4.2 is incorrect in specifying that the backports package is a dependency only on Python 2.

**Comment by minrk** [comment]
closed by #9417
